3.3_higher-order_functions.ipynb

```diff
@@ -87,7 +87,7 @@
     "\n",
     "---\n",
     "# Functions as Arguments\n",
-    "Formally, functions like `map` and `reduce` are called _higher-order functions_: they are functions that take functions as arguments.\n",
+    "Formally, functions like `map` and `reduce` are called _higher-order functions_ : they are functions that take functions as arguments.\n",
```

Comment:
This change isn't quite necessary, no?

Reply:
If you think it's not necessary then I can remove this commit, however it does cause a rendering problem on JupyterHub (italic fonts are not rendered and underscores are shown). This is a small hack.

Reply:
Good catch! They must have a bug somewhere...